hybrasyl/Scripting/HybrasylUser.cs

```diff
@@ -177,6 +177,12 @@ public List<HybrasylWorldObject> GetFacingObjects()
             return User.GetFacingObjects().Select(item => new HybrasylWorldObject(item)).ToList();
         }
 
+        public HybrasylMonster GetFacingMonster()
+        {
+            var facing = User.GetFacingMonster();
+            return facing != null ? new HybrasylMonster(facing) : null;
```

Comment:
I see the logic for having a `HybrasylMonster` rather than using the existing `HybrasylWorldObject` default - but similarly to comments above `GetFacingMonster` needs to be updated here to use existing functionality

Reply:
updated